x-pack/plugins/beats_management/public/components/config_list.tsx

```diff
@@ -35,22 +40,43 @@ export const ConfigList: React.SFC<ComponentProps> = props => (
       },
       {
         field: 'module',
-        name: 'Module',
+        name: props.intl.formatMessage({
+          id: 'xpack.beatsManagement.tagConfigTable.moduleColumnName',
+          defaultMessage: 'Module',
+        }),
         truncateText: false,
         render: (value: string) => {
-          return value || 'N/A';
+          return (
+            value ||
+            props.intl.formatMessage({
+              id: 'xpack.beatsManagement.tagConfigTable.moduleColumn.notAvailibaleLabel',
+              defaultMessage: 'N/A',
+            })
+          );
         },
       },
       {
         field: 'description',
-        name: 'Description',
+        name: props.intl.formatMessage({
+          id: 'xpack.beatsManagement.tagConfigTable.descriptionColumnName',
+          defaultMessage: 'Description',
+        }),
       },
       {
-        name: 'Actions',
+        name: props.intl.formatMessage({
+          id: 'xpack.beatsManagement.tagConfigTable.actionsColumnName',
+          defaultMessage: 'Actions',
+        }),
         actions: [
           {
-            name: 'Remove',
-            description: 'Remove this config from tag',
+            name: props.intl.formatMessage({
+              id: 'xpack.beatsManagement.tagConfigTable.actions.removeButtonAriaLabel',
+              defaultMessage: 'Remove',
+            }),
+            description: props.intl.formatMessage({
+              id: 'xpack.beatsManagement.tagConfigTable.actions.removeDescription',
```

Comment:
Also I updated `.removeDescription` to `.removeTooltip` because this label is displayed as a tooltip for button.

![image](https://user-images.githubusercontent.com/11524637/48769593-1a63d080-eccd-11e8-844a-239aacc8e5ea.png)